dir2/file1.txt

```diff
@@ -7,3 +7,12 @@ of a multiline change.
 Second paragraph also part of the second PR.
 
 Another line part of the secodn PR
+
+More changes in dir2 to match the changes in dir1 as requested
+buy cumulus5. Keep in mind that this is different change.
+
```

Comment:
This is not what I asked. Please consider the following
- Item1
- Item2
- Item3 is `something specific`

This is how it is done in markup